tasks/linter.py

```diff
@@ -181,7 +188,14 @@ def go(
         include_sds=include_sds,
     )
 
-    success = process_module_results(flavor=flavor, module_results=lint_results)
+    with section('Linter failures', collapsed=True):
+        success = process_module_results(flavor=flavor, module_results=lint_results)
+
+    with section('Linter execution time'):
+        print(color_message('Execution time summary:', 'bold'))
+
+        for e in sorted(execution_times):
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Why sort ? It seems more intuitive to list them in the order they took place